app/main.py

```diff
@@ -1,8 +1,61 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        if not 1 <= comfort_class <= 7:
+            raise "Comfort class must be between 1 and 7."
+        if not 1 <= clean_mark <= 10:
+            raise "Clean mark must be between 1 and 10."
```

Comment:
Error: Raising exceptions with string literals is incorrect. Use `raise ValueError('Clean mark must be between 1 and 10.')` instead.